app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,67 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>PrivateEvent</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Private Event</title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag "https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" %>
   </head>
 
   <body>
+  	<% if flash[:notice] %>
+  		<div class="notification is-primary global-notification">
+  			<p class="notice"><%= notice %></p>
+  		</div>
+  	<% end %>
+
+  	<% if flash[:alert] %>
+  		<div class="notification is-danger global-notification">
+  			<p class="alert"><%= alert %></p>
+  		</div>
+  	<% end %>
+
+  	<nav class="navbar is-primary is-light">
+  		<div class="navbar-brand">
+  		<%= link_to root_path, class: "navbar-item" do %>
+  			<h1 class="title is-4 has-text-white">Private Events</h1>
+  		<% end %>
+			<div class="navbar-burger burger" data-target="navbarExample">
+					<span></span>
+					<span></span>
+					<span></span>
+  		</div>
+  	 </div>
+
+			<div id="navbarExample" class="navbar-menu">
+				<div class="navbar-end">
+          <div class="navbar-item">
+            <div class="field is-grouped">
+              <p class="control">
+                <%= link_to 'Create Event', new_event_url, class: "button is-primary is-inverted" %>
+              </p>
+              <% if user_signed_in? %>
+                <p class="control">
+                  <%= link_to current_user.username, edit_user_registration_path, class: "button is-primary" %>
+                </p>
+                <p>
+                  <%= link_to "Logout", destroy_user_session_path, method: :delete, class:"button is-primary" %>
+                </p>
+              <% else %>
+              <p class="control">
+                <%= link_to 'Sign In', new_user_session_path, class: "button is-primary" %>
+              </p>
+              <p class="control">
+                <%= link_to 'Sign Up', new_user_registration_path, class: "button is-primary" %>
+              </p>
```

Comment:
- [ ] [Required] Kindly remove this logic from the views, you can always use helper methods for all of the view methods